Automattic/wp-calypso
# Add server-friendly content sanitizer for Plugins SSR

#### Proposed Changes

The current content sanitizer for Plugins doesn't work on the server. This implements a similar API that works on the server using a 3rd party node module:`jsdom`.

#### Testing Instructions

* Log out and disable JavaScript
* Go to /plugins
* You should still be able to see the Plugins landing page with the 3 sections Paid, Editor's Pick, and Free
* The unit tests from `client/lib/plugins/sanitize-section-content.js` should pass

* Enable JavaScript
* Visit a few Plugin Details page to check the content is still shown
